data/remnant missions.txt

```diff
@@ -493,5 +598,761 @@ mission "Remnant: Technology Available"
+			`	It looks like the person behind the desk has been expecting you. They gesture very slowly, asking how your lessons are going using very basic signs and songs. After a few minutes of basic conversation, they ask you for the hand-scan and then give you a datachip with more lessons.`
+			`	There is also a note saying that there will be a final package of lessons waiting for you on Viminal.`
+
+	on complete
+		conversation
+			`The trip to Viminal is quiet - Almost no talking or singing of any kind, almost entirely reliant on interposing signs and pantomiming actions. Slowly you start picking up some of the easier nouns and verbs.`
+			`	As you descend through the atmosphere you do some quick hand stretches to work the stiffness out of your fingers and rehearse some of the more common phrases. As soon as <ship> is parked you exit and look around and approach a dock hand who doesn't appear to be too busy and carefully sign "Can you direct me to the port director?" He looks at you curiously for a moment and replies with exaggerated slowness. You thank him and head towards the spaceport following his directions.`
+
+			
+mission "Remnant: Learning Sign 2"
+	name "Learning Remnant Sign"
+	description `Return to Caelian to finish your first class of sign language`
+	source "Viminal"
+	destination "Caelian"
+	to offer
+		has "license: Remnant"
+		has "Remnant: Learning Sign 1: done"
+	on offer
+		conversation
+			`Once again the director seems to be expecting you, and has a simple conversation with you about how the flight was and how your lessons are progressing. You struggle to string the sentences together, but you successfully manage to hold your own for several minutes before he directs you to place your hand on the scanner. After checking the readout he hands you another datachip as well as a stack of holographic print-outs.`
+			`	Flipping through the stack of print-outs, you note that each one of them has a small label in the corner for random pieces of furniture and ship equipment, and when you view the hologram while moving from side to side the figure inside makes a sign that you assume to be what the label indicates.`
+			choice
+				`	Accept them`
+					goto ContinueSign
+				`	Not interested after all.`
+					decline
+			label "ContinueSign"
+			`	Back on your ship you find a roll of tape and stick up all the labels to the appropriate things around your ship. Now every time you look around your ship, you see figures demonstrating how to sign things. It kind of reminds you of being back in preschool. It does seem to be working, though.`
+				accept
+	
+	on complete
+		log "You have learned the basics of the Remnant sign language, and been formally granted access to their worlds after having had your genetics tested on each of them."
```

Comment:
Logbook text don't use pronouns referring to the player.

Reply:
So more along the lines of: 
"Have learned the basics of the Remnant sign language, and been formally granted access to their worlds after having been genetically tested on each of them." ?